content/articles/2018-07-talk-carpe-diem.markdown

```diff
@@ -0,0 +1,82 @@
+---
+title: Talk Carpe Diem
+author: Jenny Bryan and Mara Averick
+date: '2018-07-06'
+slug: talk-carpe-diem
+description: >
+  Seize the day and show off your work!
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/iIWCjgK3704
+  author: Karine Germain
+---
+
+Summer conference season is in full swing, with [useR! 2018 in Brisbane](https://user2018.r-project.org) coming up next week!
+
+Conference talks are a great opportunity to help people learn about the cool and useful things you have built. A few concrete practices will make your marketing efforts more effective.
+
+*We assume your product meets your quality standards and that you have already documented it. This is about stuff you can do in ~1 hour as you prepare you talk that will have high payoff in terms of recruiting users. This is also not about how to give a great talk.*
+
+## Marketing is not a Four Letter Word
+
+First, let's deal with the word and concept of "marketing".
+
+> Marketing is the study and management of exchange relationships.
+
+https://en.wikipedia.org/wiki/Marketing
+
+If you have taken the trouble to build an R package, Shiny app, tutorial, or talk, presumably you would like people to know about it and use it! Marketing is the set of activities that help your target audience
+
+  * discover that your thing exists
+  * learn how your thing could benefit them
+
+Use your talk (and documentation and README, etc.) to accomplish these goals!
+
+Some creators take a peculiar pride in being "bad at marketing", but it is not some elusive skill. Nor is marketing ability anti-correlated with creating a high-quality product. You can acknowledge the value in marketing or you can declare that cultivating broader usage is not your goal. Pick one.
+
```

Comment:
Point to some general marketing advice? I like https://medium.com/indeed-data-science/marketing-for-data-science-a-7-step-go-to-market-plan-for-your-next-data-product-60c034c34d55 (written by Roger's son)